spec/controllers/platform/api/v1/accounts_controller_spec.rb

```diff
@@ -42,29 +42,24 @@
         json_response = JSON.parse(response.body)
         expect(json_response['name']).to eq('Test Account')
         expect(json_response['locale']).to eq('es')
-        expect(json_response['enabled_features']).to eq(
```

Comment:
I believe this line is intentional and shouldn't be removed. We want to make sure that the default features enabled by installation config should work when an account is created with out any specification through `features` params